<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimonMainWindow</class>
 <widget class="QMainWindow" name="SimonMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimonMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>120</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="blueButton">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="yellowButton">
       <property name="text">
        <string>Yellow</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="greenButton">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="newGameButton">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>New Game</string>
       </property>
       <property name="autoRepeatInterval">
        <number>101</number>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="redButton">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="instructions">
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="gameOverLabel">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>260</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="lives">
      <property name="text">
       <string>Lives</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="life1">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="life2">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="life3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="levelLabel">
         <property name="text">
          <string>Level</string>
         </property>
        </widget>
        <widget class="QLabel" name="numOfLevel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QLCDNumber" name="timer"/>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
